gridfs/grid_file.py

```diff
@@ -465,6 +465,7 @@ def __init__(self, root_collection, file_id=None, file_document=None,
 
         root_collection = _clear_entity_type_registry(root_collection)
 
+        self.__closed = False
```

Comment:
Do we need a `super().__init__()` call here now that we inherit from IOBase?

Reply:
Fixed